ivy/functional/backends/torch/sorting.py

```diff
@@ -20,3 +20,6 @@ def sort(
 ) -> torch.Tensor:
     sorted_tensor, _ = torch.sort(x, dim=axis, descending=descending, out=out)
     return sorted_tensor
+
+
+sort.unsupported_dtypes = ("uint16", "uint32", "uint64")
```

Comment:
`torch` does not support any of these data types at all, and so this line is unnecessary